### DISCUSSION TITLE
5G SA deployment issue

### DISCUSSION THREAD

**Discussion by ajaydas03** [herlesupreeth/docker_open5gs#343] [Q&A] [answered]
Hi Team,

while running the "docker compose -f sa-deploy.yaml up" comman i am getting the below error.

openshift@openshift:~/IMS-IMS/docker_open5gs$ sudo docker compose -f sa-deploy.yaml up
WARN[0000] /home/openshift/IMS-IMS/docker_open5gs/sa-deploy.yaml: `version` is obsolete
WARN[0000] Found orphan containers ([nr_ue nr_gnb pcscf icscf scscf smsc pyhss mme sgwc sgwu hss pcrf osmomsc rtpengine dns mysql osmohl                                                    r]) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans fl                                                    ag to clean it up.
[+] Running 9/12
✔ Container scp      Created                                                                                                      0.2s
⠇ Container metrics  Recreate                                                                                                     0.9s
⠇ Container grafana  Recreate                                                                                                     0.9s
✔ Container mongo    Recreated                                                                                                    0.8s
✔ Container nrf      Created                                                                                                      0.2s
✔ Container udm      Created                                                                                                      0.2s
✔ Container ausf     Created                                                                                                      0.1s
✔ Container udr      Created                                                                                                      0.1s
⠋ Container webui    Recreate                                                                                                     0.1s
✔ Container bsf      Created                                                                                                      0.1s
✔ Container pcf      Created                                                                                                      0.1s
✔ Container nssf     Created                                                                                                      0.1s
⠋ Container amf      Creating                                                                                                     0.0s
Error response from daemon: invalid config for network docker_open5gs_default: invalid endpoint settings:

no configured subnet or ip-range contain the IP address 192.168.xx.xxx

Kindly help me to resolve the issue.

Thanks

**Comment by herlesupreeth** [comment, marked as answer]
Please do not open the same issue in multiple places. Its clearly a configuration issue on your end.
